script.js

```diff
@@ -0,0 +1,128 @@
+var memory = "0",
+	currentValue = "0",
+	resultValue = 0;
+
+const maxChar = 12;
+
+const display = document.querySelector(".display p");
+
+//printing a value on the display
+
+function addValue(digit) {
+	if (eval(currentValue) === 0 && currentValue.indexOf(".") === -1) {
+		current = digit;
```

Comment:
This assignation works when you are not using a [strict mode ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode)(`"use strict";`)

But it will make your program behave weird because is expecting for currentValue.